src/index.css

```diff
@@ -26,7 +33,9 @@ body {
   font-family: 'Roboto', 'Segoe UI', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Arial','Helvetica Neue',
     'Helvetica', -apple-system, BlinkMacSystemFont, sans-serif;
   margin: 0 auto;
-  max-width: 475px;
+  max-width: 640px;
```

Comment:
why 640?

Reply:
yeah, thanks. I set it because iphone 5 size 640×1136px. but then I understood about pixel ratio=2. So I think it should be 320px. or I can cancel my changes to 475

Reply:
ah. the same. it is just middle commit. before create a PR

Reply:
also we can use material - ui break points
xs, extra-small: 0px
sm, small: 600px
md, medium: 960px
lg, large: 1280px
xl, extra-large: 1920px

Reply:
change this back to 475px this is actually for browsers Liav the designer doesnt want it too wide. maybe it should be 475dp.

Reply:
ok